README.md

```diff
@@ -141,3 +141,9 @@ To train the model we just registered run:
 ```console
 python examples/train.py train cnn --dataset mnist --epochs 10 --hparams-set basic --hparams batch_size=64
```

Comment:
With the ``` --hparams batch_size=64``` part, please update to have more then 1 hyperparamter, e.g -```-hparams batch_size=64,learning_rate=0.1``` etc. Found this out the hardway that naively using multiple --hparams instances only passes 1, writing ```--hparams batch_size=64 learning_rate=0.1``` or ```--hparams batch_size=64, learning_rate=0.1 ```causes errors